cishouseholds/pipeline/high_level_transformations.py

```diff
@@ -174,11 +174,10 @@ def transform_cis_soc_data(df: DataFrame, join_on_columns: List[str]) -> DataFra
     """
     transform and process cis soc data
     """
+
     df = df.withColumn(
         "standard_occupational_classification_code",
-        F.when(F.substring(F.col("standard_occupational_classification_code"), 1, 2) == "un", "uncodeable").otherwise(
-            F.col("standard_occupational_classification_code")
-        ),
+        F.regexp_replace(F.col("standard_occupational_classification_code"), r"[a-zA-Z].*|^\s*$", "uncodeable"),
```

Comment:
I think this might be a decent catch-all, to get anything that contains a non-numeric character
```suggestion
        F.regexp_replace(F.col("standard_occupational_classification_code"), r".*[^0-9].*", "uncodeable"),
```

Reply:
added